docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc

```diff
@@ -0,0 +1,193 @@
+[[semantic-search-semantic-text]]
+=== Tutorial: semantic search with `semantic_text`
+++++
+<titleabbrev>Semantic search with `semantic_text`</titleabbrev>
+++++
+
+beta[]
+
+This tutorial show you how to use the semantic text feature to perform semantic
+search on your data.
+
+Semantic text simplifies the {infer} workflow by providing {infer} at ingestion
+time and sensible default values automatically. You don't need to define certain
+settings and parameters or create {infer} ingest pipelines.
+
+The recommended way to perform semantic search in the {stack} is following the
+semantic text workflow. However, if you need more control over the settings, you
+can still use the complete {infer} workflow (refer to 
+<<semantic-search-inference,this tutorial>> to review the process).
+
+This tutorial uses the `elser` service for demonstration but you can use any
+service and their supported models offered by the {infer-cap} API.
+
+
+[discrete]
+[[semantic-text-requirements]]
+==== Requirements
+
+To use `semantic_text` field type, you must have an {infer} endpoint deployed in
+your cluster using the <<put-inference-api>>.
+
+
+[discrete]
+[[semantic-text-infer-endpoint]]
+==== Create the {infer} endpoint
+
+Create an inference endpoint by using the <<put-inference-api>>:
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/sparse_embedding/my-elser-model <1>
+{
+  "service": "elser", <2>
+  "service_settings": {
+    "num_allocations": 1,
+    "num_threads": 1
+  }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+<1> The task type is `sparse_embedding` in the path as the `elser` service will
+be used and ELSER creates sparse vectors. The `inference_id` is
+`my-elser-model`.
+<2> The `elser` service is used in this example.
+
+
+[discrete]
+[[semantic-text-index-mapping]]
+==== Create the index mapping
+
+The mapping of the destination index - the index that contains the embeddings
+that the model will create based on your input text - must be created. The
+destination index must have a field with the <<semantic-text,`semantic_text`>>
+field type to index the output of the used model.
+
+[source,console]
+------------------------------------------------------------
+PUT semantic-embeddings
+{
+  "mappings": {
+    "properties": {
+      "content_inference": { <1>
```

Comment:
I wonder if using semantic_text directly would be more evident. In the end, the main goal of using semantic_text is to simplify. If users need to use two fields and copy_to, we may be defeating that purpose. WDYT?